data_model/src/fixed.rs

```diff
@@ -0,0 +1,265 @@
+//! Types used for Fixed-point operations. Uses [`fixnum::FixedPoint`].
+#![allow(variant_size_differences)]
```

Comment:
would you place this over the `FixedPointOperationError` enum if it's required there?
Is there really such a difference in variant sizes for `FixedPointOperationError`?

Reply:
> place... 
yes. 
> really such difference

23 times. I could use a `Box` `fixnum::ConversionError`.